src/server/routes/v2/featuredPackages.ts

```diff
@@ -119,4 +119,43 @@ export default async (fastify: FastifyInstance): Promise<void> => {
       "failed to delete featured package details, package may not exist",
     );
   });
+
+  //* delete all featured packages
+  fastify.delete("/", async (req, res) => {
+    const accessToken = req.headers["xxx-access-token"];
+    if (accessToken == null) {
+      res.status(401);
+      throw new Error("unauthorised");
+    }
+    if (accessToken !== API_ACCESS_TOKEN) {
+      res.status(403);
+      throw new Error("forbidden");
+    }
+
+    const featuredPackages = await packageService.find({
```

Comment:
you can probs use `packageService.update({ filters: { Featured: true }, update: { Featured: true } })` or some variation of that to update them all in one go, if that doesnt work (which it might not cos i havent looked at that code in a while), then what you have here is fine